[2021-09-28 00:48:44,583] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.say_hello 2021-09-28T00:30:00+00:00 [queued]>
[2021-09-28 00:48:44,722] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.say_hello 2021-09-28T00:30:00+00:00 [queued]>
[2021-09-28 00:48:44,724] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-28 00:48:44,727] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-28 00:48:44,735] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-28 00:48:44,863] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): say_hello> on 2021-09-28T00:30:00+00:00
[2021-09-28 00:48:44,885] {standard_task_runner.py:52} INFO - Started process 25490 to run task
[2021-09-28 00:48:44,986] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'say_hello', '2021-09-28T00:30:00+00:00', '--job-id', '104', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp4jkj1hur', '--error-file', '/tmp/tmpy9yjy5ai']
[2021-09-28 00:48:45,069] {standard_task_runner.py:77} INFO - Job 104: Subtask say_hello
[2021-09-28 00:48:45,699] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.say_hello 2021-09-28T00:30:00+00:00 [running]> on host 7c1d2d2abdb5
[2021-09-28 00:48:45,993] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Veerawat
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=say_hello
AIRFLOW_CTX_EXECUTION_DATE=2021-09-28T00:30:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-28T00:30:00+00:00
[2021-09-28 00:48:45,997] {my_dag.py:11} INFO - Hello, SAKSIAM
[2021-09-28 00:48:46,002] {python.py:151} INFO - Done. Returned value was: None
[2021-09-28 00:48:46,032] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=say_hello, execution_date=20210928T003000, start_date=20210928T004844, end_date=20210928T004846
[2021-09-28 00:48:46,152] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-28 00:48:46,260] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
